Kudu.Core/Environment.cs

```diff
@@ -101,6 +104,7 @@ public class Environment : IEnvironment
             _dataPath = Path.Combine(rootPath, Constants.DataPath);
             _jobsDataPath = Path.Combine(_dataPath, Constants.JobsPath);
             _jobsBinariesPath = Path.Combine(_webRootPath, Constants.AppDataPath, Constants.JobsPath);
+            _alwaysOnEnabled = IsAlwaysOnEnabled();
```

Comment:
Basically we use it where we need it, for example: https://github.com/projectkudu/kudu/blob/master/Kudu.Core/Infrastructure/InstanceIdUtility.cs#L29
https://github.com/projectkudu/kudu/blob/master/Kudu.Services.Web/App_Start/NinjectServices.cs#L546